DEVELOPERS.md

```diff
@@ -18,16 +74,6 @@ If using VSCode for development, use these settings so that tests are discovered
 
```

Comment:
Would it be a good idea to advise the developer to include this in the local `settings.json` file _for this project_?

Reply:
Good point, I agree with you. Also, since I said at the top of the file we're assuming you're using VSCode, I removed "If using VSCode"

Resolved by 182d9f2

Reply:
I skimmed your message too quickly, I also clarified that those settings are for the *local* settings in 47bffbb

Reply:
Yes, that makes sense.